subworkflows/local/call_repeat_expansions/tests/main.nf.test.snap

```diff
@@ -37,7 +37,7 @@
                         {
                             "id": "family"
                         },
-                        "family.vcf.gz:md5,68b329da9893e34099c7d8ad5cb9c940"
+                        "family.vcf:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
We should make sure to still output compressed vcf from this subworkflow. Even in the stubs, so we can implement some stub tests in the next release.

Reply:
Gotta fix that in the module. Should've come up in the review over at nf-core/modules 🙃

Reply:
You can ping me for a review when you're done!